src/probability/moments.lean

```diff
@@ -205,6 +211,63 @@ begin
   exact log_mul (mgf_pos' hμ h_int_X).ne' (mgf_pos' hμ h_int_Y).ne',
 end
 
+lemma indep_fun.integrable_exp_mul_add {X Y : Ω → ℝ} (h_indep : indep_fun X Y μ)
+  (h_int_X : integrable (λ ω, exp (t * X ω)) μ)
+  (h_int_Y : integrable (λ ω, exp (t * Y ω)) μ) :
+  integrable (λ ω, exp (t * (X + Y) ω)) μ :=
+begin
+  simp_rw [pi.add_apply, mul_add, exp_add],
+  exact (h_indep.exp_mul t).integrable_mul h_int_X h_int_Y,
+end
+
+lemma Indep_fun.integrable_exp_mul_sum [is_probability_measure μ]
+  {X : ι → Ω → ℝ} (h_indep : Indep_fun (λ i, infer_instance) X μ) (h_meas : ∀ i, measurable (X i))
+  {s : finset ι} (h_int : ∀ i ∈ s, integrable (λ ω, exp (t * X i ω)) μ) :
+  integrable (λ ω, exp (t * (∑ i in s, X i) ω)) μ :=
+begin
+  classical,
+  revert h_int,
+  refine finset.induction_on s (λ h, _) (λ i s hi_notin_s h_rec h_int, _),
```

Comment:
The induction tactic will do the reverting for you:
```suggestion
  induction s using finset.induction_on with i s hi_notin_s h_rec h_int,
```
Same thing in other proofs below.